src/main/java/org/jitsi/videobridge/rest/JSONDeserializer.java

```diff
@@ -326,39 +360,40 @@ public static void deserializeChannelCommon(
 
         // id
         if (id != null)
+        {
             channelIQ.setID(id.toString());
+        }
         // channelBundleId
         if (channelBundleId != null)
+        {
             channelIQ.setChannelBundleId(channelBundleId.toString());
+        }
         // endpoint
         if (endpoint != null)
+        {
             channelIQ.setEndpoint(endpoint.toString());
+        }
         // expire
         if (expire != null)
         {
-            int i;
+            int i = objectToInteger(expire);
```

Comment:
wish this was named something more useful, i see it was existing code but it'd be nice to clean it up.

Reply:
Do you mean "i" or "objectToInteger"? I've removed the "i" as you suggested above.